### ISSUE TITLE
Feature: Deadzone control for both ends of axis

### ISSUE THREAD

**Issue by diVineProportion** [Snoothy/UCR#46]
Any chance for something like this:
![image](https://user-images.githubusercontent.com/38984875/46793120-ce029980-ccf9-11e8-8059-cbf1d7c4eb70.png)
In both UCR and UCR-C you can only control the deadzone around the center of the axis correct?

**Comment by evilC** [comment]
What code is used for each plugin to handle deadzones is dictated by the plugin itself.  
There would be nothing stopping you from writing new code, and making a plugin use that code instead for deadzones.  
[Plugin calling DZ function](https://github.com/Snoothy/UCR/blob/master/UCR.Plugins/Remapper/AxisToAxis.cs#L45)  
[Existing DeadzoneHelper class](https://github.com/Snoothy/UCR/blob/master/UCR.Core/Utilities/AxisHelpers/DeadZoneHelper.cs)

So you could write an alternative DeadzoneHelper, then make a plugin that uses it.

**Comment by evilC** [comment]
I am against anything using sliders, period.  
UNLESS:  
There is an equivalent text box to type exact value.  
OR  
You can use eg arrow keys to select exact value.  
Otherwise, unless the slider is over 100px wide, it's impossible to select all values, and even then it's a right pain to select the exact value you want.

**Comment by evilC** [comment]
A thought - given that the DZ on each of the scales is identical, a linear sensitivity option of <100% would achieve the same result.

**Comment by diVineProportion** [comment]
I completely agree about having the text box to help hone in on the specific value.

The DZ are not necessarily identical on each end. I'll give you an example. First I will point out that the GUI I referenced in the OP was from the Saitek (now under Logitech) X52 Professional and I didn't think to show one of it's other features. If you right clicked in the one of the DZ areas where you see the current DZ value, you get this: 

![image](https://user-images.githubusercontent.com/38984875/51083683-dbea0100-16d2-11e9-817c-ff922b8c7762.png)

So the identical linked DZ's work great for something like my RX axis seen here
![zxfievvepv](https://user-images.githubusercontent.com/38984875/51083758-efe23280-16d3-11e9-9ee2-8b294e4c63ef.gif)

But what about the case of my throttle quadrant where physically I have each of the axis set at 50% (Throttle, Pitch and Yaw are all equal in straight line right at 50%), but logically they appear like this: 
![image](https://user-images.githubusercontent.com/38984875/51083770-36379180-16d4-11e9-8223-a791de1bb775.png)

If I didn't have the ability to unlink the DZ's, I'd be forced to sacrifice axis positions as seen here

![image](https://user-images.githubusercontent.com/38984875/51083781-7565e280-16d4-11e9-9fd5-3cfdeccbaddf.png)

Whereas unlinking the axes would allow: 
![image](https://user-images.githubusercontent.com/38984875/51083803-d097d500-16d4-11e9-955d-9468d8632653.png)

You may be wondering why exactly i care to do this when they are not logically at 50%. The thing is that it isn't important what they are at, rather that I know i can physically put each axis at the physical 50% mark, and I know that they will be residing in a DZ and not interfere etc,
